README.md

```diff
@@ -1,39 +1,60 @@
 # "Hello world"? We prefer "Hello Bao!"
 
-Welcome to the Bao Hypervisor! Get ready for an interactive journey as we explore the world of Bao together. Whether you're a seasoned Bao user or a newcomer, this tour is designed to give you a practical and enthusiastic introduction to our powerful hypervisor.
+Welcome to the Bao Hypervisor! Get ready for an interactive journey as we
+explore the world of Bao together. Whether you're a seasoned Bao user or a
+newcomer, this tour is designed to give you a practical and enthusiastic
+introduction to our powerful hypervisor.
 
-If you're already familiar with Bao or want to dive into specific setups provided by our team, feel free to skip ahead to the Bao demos section.
+If you're already familiar with Bao or want to dive into specific setups
+provided by our team, feel free to skip ahead to the Bao demos section.
 
-In this guide, we will take a tour of the different components required to build a setup using the Bao hypervisor and learn how the different components interact. For this purpose, the guide contains the following topics:
+In this guide, we will take a tour of the different components required to
+build a setup using the Bao hypervisor and learn how the different components
+interact. For this purpose, the guide contains the following topics:
 
-- A **getting started** to help users on preparing the environment to build the setup and also some pointers to documentations of Bao (in case you want to go deeper in any detail);
+- A **getting started** to help users on preparing the environment to build the
+  setup and also some pointers to documentations of Bao (in case you want to go
+  deeper in any detail);
 
-- An **initial setup** for giving the first steps on this tour. This section aims to explore the different components of the system and get the first practical example of this guide;
+- An **initial setup** for giving the first steps on this tour. This section
+  aims to explore the different components of the system and get the first
+  practical example of this guide;
 
 - An **interactive tutorial on changing the guests** running on top of Bao;
 
 - A **practical example** of changing the setup running;
 
-- An example of **how different guests can coexist and interact** with each other;
+- An example of **how different guests can coexist and interact** with each
+  other;
```

Comment:
Remove the newlines between the items of the list. Extra space is being used.

Reply:
Done!